doc/dev_guide/internals/box_protocol.rst

```diff
@@ -208,25 +215,30 @@ Almost all requests and responses contain three sections: size, header, and body
 The size is an (MP_UINT) unsigned integer, usually a 32-bit unsigned integer.
 The header and body are (MP_MAP) maps.
 
-:code:`# <size>` |br|
-:samp:`{MP_UINT unsigned integer}` |br|
-:code:`# <header>` |br|
-:samp:`{MP_MAP with <header> map-items}` |br|
-:code:`# <body>` |br|
-:samp:`{MP_MAP with <body> map-items}` |br|
+.. code-block:: c
```

Comment:
Use [`:emphasize-lines:`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-option-code-block-emphasize-lines) to emphasize lines.